### PR TITLE
[res] TensorFlowLiteRecipes for Softmax decomposition

### PR DESCRIPTION
**Pull request by stamalakhov**
This commit adds two models for Softmax with unit/nonunit beta.
Its correctness is tested at #11687.

Draft: #11687
Related: #11492
ONE-DCO-1.0-Signed-off-by: s.malakhov <s.malakhov@partner.samsung.com>

### CODE REVIEW

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by jinevening — APPROVED** [review]
LGTM
